phenol-cli/pom.xml

```diff
@@ -27,6 +27,12 @@
       <version>${jcommander.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
Is this being used, or is JCommander?

Reply:
not sure what you mean by "this" here. In general, I am migrating from JCommander to picocli, and that would be good here in the futre.